RoeiJeRot.Logic/PasswordHasher.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+
+namespace RoeiJeRot.Logic
+{
+    class PasswordHasher
```

Comment:
Laten we overal comments doen op public types en functions. Had dit nog niet gezegd maar maakt het over het algemeen veel netter en las het ook in de beoordeling ergens.